contributing/releasing.md

```diff
@@ -62,6 +62,12 @@ Note: if for some reason `cut` fails, first ensure that nobody else is in the mi
 You will now have a local `release-candidate` branch, a new section in CHANGELOG.md titled
 "release-candidate", and the `release-candidate` branch will have been pushed to GitHub.
 
+#### Create the PR fore the PR
```

Comment:
```suggestion
#### Create the PR for the PR
```

Reply:
I'm sad that I don't have access to the beta for Suggestions!  I can find no documentation about how to open a suggestion...

Reply:
It's super unintuitive, but it's this button in the in-line comment form:

<img width="44" alt="screen shot 2018-12-05 at 12 47 55 pm" src="https://user-images.githubusercontent.com/45670/49533059-06eb6480-f88c-11e8-87dd-df29038bf812.png">

Reply:
Let me know if you figure out how to join the beta. I found the instructions here.

https://help.github.com/articles/incorporating-feedback-in-your-pull-request/#applying-a-suggested-change

Reply:
Ahhhh.... I see it. I can even add one here! Looks like it's just a "code block" in the `suggestion` language.

``suggestion
#### Create the PR fore the PR
``